app/assets/stylesheets/webapp/webviews.scss

```diff
@@ -315,6 +315,11 @@ body.webapp {
     text-decoration: underline;
     color: #0C75E1;
   }
+
+  .button_to input.login-link{
+    color: #ffffff; 
+  }
+
```

Comment:
This placement is by no means bad, but would it make more sense in the header-specific styles? Also, should we use `@white`?